bin/martinize2

```diff
@@ -912,19 +912,6 @@ def entry():
         write_canon=args.write_canon,
     )
 
-    # grompp has a limit in the number of character it can read per line
-    # (due to the size limit of a buffer somewhere in its implementation).
-    # The command line can be longer than this limit and therefore
-    # prevent grompp from reading the topology.
-    gromacs_char_limit = 4000  # the limit is actually 4095, but I play safe
-    command = " ".join(sys.argv)
-    if len(command) > gromacs_char_limit:
-        command = command[:gromacs_char_limit] + " ..."
-    system.meta["header"].extend(("This file was generated using the following command:",
-                                  command,
-                                  VERSION,
-                                 ))
```

Comment:
I should've been more clear I think: add the full string to the header list here, but do the length limiting in the topology writer (for *all* headers/comments)